src/tasks/WasmAppBuilder/PInvokeTableGenerator.cs

```diff
@@ -88,7 +103,7 @@ void CollectPInvokesForMethod(MethodInfo method)
                 string? signature = SignatureMapper.MethodToSignature(method);
                 if (signature == null)
                 {
-                    throw new LogAsErrorException($"Unsupported parameter type in method '{type.FullName}.{method.Name}'");
+                    throw new NotSupportedException($"Unsupported parameter type in method '{type.FullName}.{method.Name}'");
```

Comment:
Why the change? This would show up as an exception with the trace, instead of a neat error message. The trace won't give any extra information, AFAICS.

Reply:
The exception is caught above. This it gets the same wrapper text as other exceptions.